docs/migrate-istio-to-asm/traffic-split/service-entry.yaml

```diff
@@ -0,0 +1,16 @@
+# With this ServiceEntry, we're instructing the mesh to
+# direct HTTP requests (from the mesh) for v2.example.com to the specified IP address.
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
```

Comment:
region tag (if you wanted to show the yaml file?)

Reply:
region tag would be: `servicemesh_traffic_spli_service_entry_serviceentry_external_service_http`

Reply:
Done. ✅  Thanks for formulating the tag for me.